docs/tutorials/add-oauth2-provider.md

```diff
@@ -173,9 +169,34 @@ class [PROVIDER NAME] extends OAuth2
 
 Please mention in your documentation what resources or API docs you used to implement the provider's OAuth2 protocol.
 
-## 3. Test your provider
+## 3. Add provider to console
+
+### 3.1 Add provider logo
+
+To add the logo for the provider you are implementing, you will need to create a separate pull request in [`appwrite/console`](https://github.com/appwrite/console) repository.
+
+Appwrite requires 4 copies of the logo, 2 in color and 2 in grayscale, all in SVG format in the following locations.
+
+```bash
+ 
+static/icons/dark/color/XXX.svg
+ 
+static/icons/dark/grayscale/XXX.svg
+
+static/icons/light/color/XXX.svg
+
+static/icons/light/grayscale/XXX.svg
+
+```
+> Where `XXX` is the name of the provider in `lowercase`.
+
+### 3.2 Add provider doc
+
+Make sure you update the documentation url for the implemented provider in `oauth-providers.ts` located in the [`src/lib/stores/oauth-providers.ts`](https://github.com/appwrite/console/blob/main/src/lib/stores/oauth-providers.ts) in the [`appwrite/console`](https://github.com/appwrite/console) repository.
+
+## 4. Test your provider
 
-After you finished adding your new provider to Appwrite, you should be able to see it in your Appwrite console. Navigate to 'Project > Users > Providers' and check your new provider's settings form.
+To test your provider within appwrite make sure you make the changes required for the console in the `console` git submodule present at [`/app/console`](https://github.com/appwrite/appwrite/tree/master/app) for testing purposes. Navigate to **Auth > Setting > OAuth2 Providers** in your appwrite instance to find the newly added provider and test it out.
```

Comment:
We need to ask the reader to update the branch for the console in the `.gitmodules` folder https://github.com/appwrite/appwrite/blob/master/.gitmodules#L4 

And then run `git submodule update --init` from the root of the project.

Following which they need to `cd app/console` and run `git pull origin <branch-name>`